### PR TITLE
HPM techs, inventions & rebels

### PR DESCRIPTION
**Pull request by cetvrtak**

### CODE REVIEW

File: EU4toV2/Source/Mappers/StartingTechMapper/StartingTech.cpp [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
-			name = nameStr.getString();
-		});
-	registerKeyword("score", [this](const std::string& unused, std::istream& theStream)
-		{
-			const commonItems::singleDouble scoreDbl(theStream);
-			score = scoreDbl.getDouble();
-		});
-	registerRegex("[a-zA-Z0-9\\_.:]+", commonItems::ignoreItem);
+	registerKeyword("group", [this](const std::string& unused, std::istream& theStream) {
+		group = commonItems::singleString(theStream).getString();
```
Comment (Idhrendur) [inline EU4toV2/Source/Mappers/StartingTechMapper/StartingTech.cpp:9]:
```suggestion
		group = commonItems::getString(theStream);
```

File: EU4toV2/Source/Mappers/StartingTechMapper/StartingTech.cpp [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
+	});
+	registerKeyword("name", [this](const std::string& unused, std::istream& theStream) {
+		name = commonItems::singleString(theStream).getString();
```
Comment (Idhrendur) [inline EU4toV2/Source/Mappers/StartingTechMapper/StartingTech.cpp:12]:
```suggestion
		name = commonItems::getString(theStream);
```

File: EU4toV2/Source/Mappers/StartingTechMapper/StartingTech.cpp [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
+	});
+	registerKeyword("hpm", [this](const std::string& unused, std::istream& theStream) {
+		hpm = commonItems::singleString(theStream).getString();
```
Comment (Idhrendur) [inline EU4toV2/Source/Mappers/StartingTechMapper/StartingTech.cpp:15]:
```suggestion
		hpm = commonItems::getString(theStream);
```

File: EU4toV2/Source/Mappers/StartingTechMapper/StartingTech.cpp [hunk 4]
```diff
@@ -1,25 +1,29 @@
...
+	});
+	registerKeyword("score", [this](const std::string& unused, std::istream& theStream) {
+		score = commonItems::singleDouble(theStream).getDouble();
```
Comment (Idhrendur) [inline EU4toV2/Source/Mappers/StartingTechMapper/StartingTech.cpp:18]:
```suggestion
		score = commonItems::getDouble(theStream);
```

**Review by Idhrendur — APPROVED** [review]

**Review by Zemurin — APPROVED** [review]
Aside those lambdas, everything else looks good.

**Pull request merged**